### PR TITLE
Make `django` depend on `redis` in local Docker 

### PR DESCRIPTION
**Pull request by PoDuck**
## Description

The django container needs to depend on redis in both local and production docker-compose files.  This caused me some confusion while implementing services that depended on redis, so I decided to fix it.

Checklist:

- [ ] I've made sure that `tests/test_cookiecutter_generation.py` is updated accordingly (especially if adding or updating a template option)
I ran the tests, and it broke nothing.  No new tests needed.
- [ ] I've updated the documentation or confirm that my change doesn't require any updates
This should not need any changes in the documentation.

### CODE REVIEW

File: {{cookiecutter.project_slug}}/local.yml [hunk 1]
```diff
@@ -13,6 +13,7 @@ services:
     container_name: django
     depends_on:
       - postgres
+      - redis
```
Comment (Andrew-Chen-Wang) [inline {{cookiecutter.project_slug}}/local.yml:16]: This is only necessary if celery is installed

**Review by Andrew-Chen-Wang — CHANGES_REQUESTED** [review]

File: {{cookiecutter.project_slug}}/local.yml (hunk 1 above)
Comment (browniebroke) [reply {{cookiecutter.project_slug}}/local.yml:16]: Agree, there is no redis container if Celery isn't used.

**Review by browniebroke — APPROVED** [review]
Should be good to go now

**Comment by browniebroke** [comment]
Thanks 🎉

**Pull request merged**